### ISSUE TITLE
Is there a way to query the state of the YX5300?

### ISSUE THREAD

**Issue by scottchiefbaker** [bluejunimo/YX5300_ESP32#2]
Is there a way to query the state of the MP3 chip? It would be helpful to know if it's currently: playing, paused, stopped, etc.

I played an entire song and (I assume) it's now stopped. `queryCurrentTrackIndex()` still reports that it's on the same index. If it's not actively playing I kind of expected that to be 0?

**Comment by bluejunimo** [comment]
If i remember from making the library, the current state query never worked properly, so i never implemented it. I'll try again though, no promises though :)

**Comment by bluejunimo** [comment]
I'm having some trouble interfacing with my YX5300s at the moment, so I cannot test the new function queryDeviceState()

**Comment by scottchiefbaker** [comment]
OK thanks for testing. Let me know what you find. I love this library and I want to help make it even better.

**Comment by bluejunimo** [comment]
I've ordered some more modules as I believe the ones I have are broken. I'll be able to test things again in a month when I receive them.

**Comment by scottchiefbaker** [comment]
Did your modules arrive? I'd be curious to work together and make some minor improvements on this.

**Comment by bluejunimo** [comment]
Glad you're still interested! I got them early september with some new ESP C6s, and I managed to get it playing. I can't remember if I got Serial communication between the computer and the ESP working or not in tadem with music playing, but I remember getting stuck on figuring out the queries. i'm back in school now so it's been difficult making time to come back to it but I do have some time this weekend so i'll see if I can take a crack at it again.
